packages/yew/src/virtual_dom/vtag.rs

```diff
@@ -363,9 +363,20 @@ impl VTag {
     /// Not every attribute works when it set as an attribute. We use workarounds for:
     /// `value` and `checked`.
     pub fn add_attribute(&mut self, key: &'static str, value: impl Into<AttrValue>) {
-        self.attributes
-            .get_mut_index_map()
-            .insert(AttrValue::Static(key), value.into());
+        self.attributes.get_mut_index_map().insert(
+            AttrValue::Static(key),
+            (value.into(), ApplyAttributeAs::Attribute),
+        );
+    }
+
+    /// Set the given key as property on the element
+    ///
+    /// [`js_sys::Reflect`] is used for setting properties.
+    pub fn add_property(&mut self, key: &'static str, value: impl Into<AttrValue>) {
```

Comment:
In the future, we should allow properties to be anything `Into<JsValue>`, but `Into<AttrValue>` is fine for this pull request.

Reply:
This sounds like opening gates for a disaster. `onclick` and `~onclick` have different meanings then. The former is handled through Yew while the other would completely bypass Yew (and all the bubbling, etc) and apply directly to the element, all without use of noderef. Unless someone comes with a genuine need for this, I wouldn't want to add this